include/dxc/HLSL/HLOperations.h

```diff
@@ -41,7 +41,17 @@ enum class HLOpcodeGroup {
   HLMatLoadStore,
   HLSelect,
   HLCreateHandle,
+  // FIXME: Change the way these groups are being proliferated/used for each
```

Comment:
Do we have an issue tracking this?

Reply:
I have added it to issue https://github.com/microsoft/DirectXShaderCompiler/issues/5699.